### ISSUE TITLE
Grafana logs a lot of warns/errors since PR 54208

### ISSUE THREAD

**Issue by dfoxg** [grafana/grafana#55654]
**What happened**:

Since the PR https://github.com/grafana/grafana/pull/54208 I get a lot of following error logs:
```
2022-09-20 20:45:26 | logger=http.server t=2022-09-20T18:45:26.284627046Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:delete scope=annotations:type:organization
2022-09-20 20:45:26 | logger=http.server t=2022-09-20T18:45:26.284537379Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:create scope=annotations:type:organization
2022-09-20 20:45:05 | logger=http.server t=2022-09-20T18:45:05.980717944Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:write scope=annotations:type:organization
2022-09-20 20:45:05 | logger=http.server t=2022-09-20T18:45:05.980151627Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:delete scope=annotations:type:organization
2022-09-20 20:45:05 | logger=http.server t=2022-09-20T18:45:05.980084789Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:create scope=annotations:type:organization
2022-09-20 20:44:52 | logger=http.server t=2022-09-20T18:44:52.907412585Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:write scope=annotations:type:organization
2022-09-20 20:44:52 | logger=http.server t=2022-09-20T18:44:52.90738551Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:delete scope=annotations:type:organization
2022-09-20 20:44:52 | logger=http.server t=2022-09-20T18:44:52.907259236Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:create scope=annotations:type:organization
2022-09-20 20:44:24 | logger=http.server t=2022-09-20T18:44:24.477130023Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:write scope=annotations:type:organization
2022-09-20 20:44:24 | logger=http.server t=2022-09-20T18:44:24.47711997Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:delete scope=annotations:type:organization
2022-09-20 20:44:24 | logger=http.server t=2022-09-20T18:44:24.477073428Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:create scope=annotations:type:organization
2022-09-20 20:44:19 | logger=http.server t=2022-09-20T18:44:19.905081718Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:write scope=annotations:type:organization
2022-09-20 20:44:19 | logger=http.server t=2022-09-20T18:44:19.905058131Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:delete scope=annotations:type:organization
2022-09-20 20:44:19 | logger=http.server t=2022-09-20T18:44:19.905008606Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:create scope=annotations:type:organization
2022-09-20 20:44:12 | logger=http.server t=2022-09-20T18:44:12.395419483Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:write scope=annotations:type:organization
2022-09-20 20:44:12 | logger=http.server t=2022-09-20T18:44:12.39540888Z level=warn msg="Failed to evaluate permission" err="no resolver found" action=annotations:delete scope=annotations:type:organization
```

I didnt chance anything, so I think this PR is broken.

**What you expected to happen**:

No error-logs should appear. 

**How to reproduce it (as minimally and precisely as possible)**:

I dont exactly know

**Anything else we need to know?**:

**Environment**:
- Grafana version: 9.1.6
- Data source type & version: prometheus, loki
- OS Grafana is installed on: docker
- User OS & Browser: Windows 11 / Edge
- Grafana plugins:
- Others:

**Comment by gamab** [comment]
Hey, thanks for reporting. I think I found the issue, I've open PRs to fix it, I'll try to get them merged by next week :+1:

**Comment by gamab** [comment]
PRs have been merged and the fix should be available in next v9.1.x (`9.1.7` I believe) :slightly_smiling_face:
